### DISCUSSION TITLE
Application Error

### DISCUSSION THREAD

**Discussion by mamiwatadev** [vercel/community#5809] [Help]
### Summary

Hi guys,

I'm technically challenged and we hired someone who's currently away on vacation to create our site. We had someone update our SEO today and suddenly the site started giving an application error. The SEO guy is at a loss and doesn't know what happened. Can one of you please help? We've been trying to get a hold of vercel and github with little help. Happy to pay for their premium support if only we can get a hold of them.

### Steps to Reproduce

NA

**Comment by amyegan** [comment]
Hey @mamiwatadev. We need to know what the error is, if you can share it here. 🙏
